gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Sun May 01 21:33:12 SAMT 2016
-distributionBase=GRADLE_USER_HOME
-distributionPath=wrapper/dists
-zipStoreBase=GRADLE_USER_HOME
-zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.13-bin.zip
+#Sun Sep 25 16:18:33 PDT 2016
+distributionBase=GRADLE_USER_HOME

```

Comment:
please, use autocrlf https://help.github.com/en/articles/configuring-git-to-handle-line-endings